### ISSUE TITLE
Failed to locate the entry point in lobechat code 

### ISSUE THREAD

**Issue by wesamoyo** [lobehub/lobe-chat#4395]
Hey I used love hat but trying to figure out what is the start file or entry point of the app am trying to deploy on cpanel

**Comment by wesamoyo** [comment]
Yes because I used the open source code to creat a custom project of mine
but am trying to figure out the entry point of the web app as I have failed
to run it in the cpanel

**Comment by wesamoyo** [comment]
Have failed to so wanted to ask the authors

**Comment by wesamoyo** [comment]
Are you sure where is the entry point or the file located is it in the
package.json or I would to talk to the developers

**Comment by felixiho** [comment]
hi @wesamoyo 
Please note that cpanel is just a web software that helps to manage your hosting server easily. 

In other to help you deploy, we'll need to know:

1. Your hosting provider 
2. Your server type. i.e shared hosting or VPS or dedicated hosting
